functional-tests/src/test/groovy/pages/app/OrganizationsPage.groovy

```diff
@@ -0,0 +1,119 @@
+package pages
```

Comment:
This page is bigger than normal because it has to handle the Update and Delete workflows, and parts of the Add workflow.  Tried to organize it by workflow type to make things easier to read.